src/test/scala/org/kibanaLoadTest/test/KibanaAPITest.scala

```diff
@@ -72,4 +72,13 @@ class KibanaAPITest {
     val loginClosure: Executable = () => helper.loginIfNeeded(httpClient)
     assertDoesNotThrow(loginClosure, "helper.loginIfNeeded throws exception")
   }
+
+  @Test
+  def generateCookiesTest(): Unit = {
+    val cookieCount = 1000
+    val client = new KbnClient(config)
+    val cookieLst = client.generateCookies(cookieCount)
+    assertEquals(cookieCount, cookieLst.length)
+    assertTrue(cookieLst(0).startsWith("sid=Fe26.2"))
```

Comment:
Would you like to check that the cookies are distinct?

Reply:
Good point! I will add it.